src/components/src/AppHeaderLogin/appheaderlogin.main.tsx

```diff
@@ -281,12 +284,32 @@ class AppHeaderLoginMain extends Component<AppHeaderLoginMainProps, AppHeaderLog
         });
     }
 
+    fetchRequisitionListsData() {
```

Comment:
You could potentially merge this into `getAccountData()` to make a single call with multiple zooms. Then saving on the requests.

Reply:
Hey @shaunmaharaj 
Seem like I can't merge it with `getAccountData()` because it uses adminFetch

Reply:
@rostyk-kanafotskyy Ah, good point. Okay let's keep this request and we can discuss some future thoughts around it with the back-end as FYI 😄

Reply:
Thanks @shaunmaharaj 🙂